packages/relic/lib/src/body/body.dart

```diff
@@ -0,0 +1,120 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:relic/src/body/types/body_type.dart';
+import 'package:relic/src/body/types/mime_type.dart';
+
+/// The body of a request or response.
+///
+/// This tracks whether the body has been read. It's separate from [Message]
+/// because the message may be changed with [Message.copyWith], but each instance
+/// should share a notion of whether the body was read.
+class Body {
+  /// The contents of the message body.
+  ///
+  /// This will be `null` after [read] is called.
+  Stream<Uint8List>? _stream;
+
+  /// The length of the stream returned by [read], or `null` if that can't be
+  /// determined efficiently.
+  final int? contentLength;
+
+  final BodyType? contentType;
```

Comment:
Suggestion/Discussion: This is missing a description of a public field.

There are also no descriptions for the factory methods.

As a general linter rule we require descriptions for all public methods and fields, one should probably be added for all of these as well.

Reply:
I tried to add comments on all public classes/methods/fields, but this will also be done on a separate pr when we add `serverpod_lints` package